std_msgs/Header header
string frame_id
geometry_msgs/Point position
std_msgs/ColorRGBA color
